packages/core/src/components/overlay/overlay.tsx

```diff
@@ -84,6 +84,11 @@ export interface IBackdropProps {
     backdropProps?: React.HTMLProps<HTMLDivElement>;
 
     /**
+     * Whether underlying content should be scrollable while the backdrop is visible.
+     */
+    backgroundScrollingDisabled?: boolean;
```

Comment:
For Overlay, I think it should be configurable because it is a low level component. I can imagine a sidebar created with Overlay which allows scrolling on the main content. It should be set to `false` for Dialogs, though.